packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md

```diff
@@ -40,13 +36,13 @@ Identity engine provides:
 
```

Comment:
Comment on * App-level policies. Just an fyi, I understood we weren't supposed to use en-dashes or em-dashes in our docs (except in a list). That said, I like it.